extensions/guacamole-auth-json/src/main/java/org/apache/guacamole/auth/json/RequestValidationService.java

```diff
@@ -52,6 +52,7 @@ public class RequestValidationService {
      * @param confService
      *     The (mock) instance of ConfigurationService
      */
+    @Inject
```

Comment:
With this annotation present, does the `@Inject` on the `confService` member variable have any effect? Or are things now injected purely through invoking this constructor?

If only the constructor is used, then the docs for the constructor should probably be updated to capture its true usage, which is no longer specific to unit testing.

Reply:
Good point - I'll take a look at both those items and refactor.